<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отгрузка</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Склад</string>
    </property>
    <addaction name="actionShip"/>
    <addaction name="actionRet"/>
    <addaction name="actionAvEl"/>
    <addaction name="actionAvWire"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Партии</string>
    </property>
    <addaction name="actionPartEl"/>
    <addaction name="actionPartWire"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Отчеты</string>
    </property>
    <addaction name="actionReportEl"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionShip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отгрузка</string>
   </property>
  </action>
  <action name="actionRet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Возврат / перемещение электродов</string>
   </property>
  </action>
  <action name="actionPartEl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Движение электродов</string>
   </property>
  </action>
  <action name="actionPartWire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Движение проволоки</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAvEl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Наличие электродов</string>
   </property>
  </action>
  <action name="actionAvWire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Наличие проволоки</string>
   </property>
  </action>
  <action name="actionReportEl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Отчет по электродам</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
